packages/fxa-payments-server/src/images/fpn.svg

```diff
@@ -0,0 +1,30 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="96" height="96" viewBox="0 0 96 96">
```

Comment:
I would love for the team to start getting into the habit of running SVGs through something called `svgo` which compresses/optimizes SVGs for us, similar to sending a large PNG through tinyPNG. There's a command line tool you can install with `npm i -g svgo` and then you can run `svgo /path/to/file`, or you can go to https://jakearchibald.github.io/svgomg/ and open the SVG there. :) I've compressed a few in our repo and it's reduced the file size up to 50-80%.

Side note: we _could_ explore automating this, but every now and then (like 1 - 3% of the time?) this will strip out a mask or something that's needed for it to display properly so I always double check it looks right before committing.

Your PR looks good either way though. To reiterate what I said in Slack / for future reference, we'll probably end up doing something like "className={`graphic-${productName}`}" (or whatever the variable for the name is here) on the div when we have multiple products to make this dynamic.

Reply:
I think the intent is to eventually just have a generic "check" icon instead of doing something per product, but i'll svgo the svg!